client/src/components/Profile/primary.tsx

```diff
@@ -22,9 +25,18 @@ const Image = ({
   src = '/icons/default-profile-icon.svg',
   editMode,
 }: ImageProps) => {
+  const [userInfoState, setUserInfoState] = useRecoilState(userState);
+  const ref = useRef<HTMLInputElement>(null);
+  const imageRef = useRef<HTMLImageElement>(null);
```

Comment:
unused? 
how about use this(`imageRef`) instead of using `ref`?

Reply:
imageRef naming was for avoid duplicated ref names. But from now on it will not use. So I think use only ref variable name is ok. How think about you?

Reply:
First, I deleted unused codes.
etc) userInfoState, imageRef